xbmc/video/windows/GUIWindowFullScreen.cpp

```diff
@@ -31,6 +31,7 @@
 #include "video/dialogs/GUIDialogAudioSubtitleSettings.h"
 #include "guilib/GUIWindowManager.h"
 #include "input/Key.h"
+#include "pvr/PVRGUIActions.h"
```

Comment:
dependency to PVRManager is already not ideal. Adding an additional pvr dependency should be avoided.